package.json

```diff
@@ -0,0 +1,47 @@
+{
+  "name": "doc-api",
+  "version": "1.0.0",
+  "description": "This is a document api",
+  "main": "index.js",
+  "directories": {
+    "test": "test"
+  },
+  "scripts": {
+    "pretest": "node server/seeder/seeds.js",
+    "test": "istanbul cover ./node_modules/mocha/bin/_mocha test/server/*.test.js && cat ./coverage/lcov.info | ./node_modules/coveralls/bin/coveralls.js",
+    "start": "node server.js"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/andela-ekahindi/doc-api.git"
+  },
+  "author": "Esther Dama Kahindi",
+  "license": "MIT",
+  "bugs": {
+    "url": "https://github.com/andela-ekahindi/doc-api/issues"
+  },
+  "homepage": "https://github.com/andela-ekahindi/doc-api#readme",
+  "dependencies": {
+    "async": "^2.0.1",
+    "bcrypt-nodejs": "0.0.3",
+    "body-parser": "^1.15.2",
+    "chai": "^3.5.0",
+    "eslint": "^3.5.0",
+    "eslint-config-airbnb-base": "^7.1.0",
+    "eslint-plugin-import": "^1.14.0",
```

Comment:
ESLint and its config/plugin packages are better saved as devDependencies since they are not needed outside the dev environment.